devicetypes/smartthings/aeon-key-fob.src/aeon-key-fob.groovy

```diff
@@ -56,13 +56,25 @@ metadata {
 
 def parse(String description) {
 	def results = []
+
+	if (!device.currentState("supportedButtonValues")) {
```

Comment:
You could put this in `buttonEvent` instead of `parse` because we do a similar check for `numberOfButtons` in `buttonEvent`, and plus it would mean this check would only happen on button presses and not on every message.

Reply:
I thought about that. My concern is that the change is related to something that, if mobile doesn't get my requested change for rule builder's behavior in for the upcoming release, one won't be able to use the device in automations until this event is sent.

Maybe what can be done is that I'll follow this PR up with a PR to move that code into the button handler when the next release is ready? Or, I suppose I could just put it in the WakeUpNotification handler -- I assume all devices send it?

Reply:
OK that makes sense. It would be nice to remove this eventually. Perhaps it could be moved to `updated` after a few weeks so it hasn't taken effect for some reason (dead battery) the user at least has a way to manually recover it.